src/megameklab/com/printing/PrintRecordSheet.java

```diff
@@ -297,7 +297,7 @@ public int print(Graphics graphics, PageFormat pageFormat, int pageIndex) {
             createDocument(pageIndex, pageFormat);
             GraphicsNode node = build();
             node.paint(g2d);
-            /* Testing code that outputs the generated svg
+            /* Testing code that outputs the generated svg */
```

Comment:
Is this test code still?

Reply:
Oops, missed that. I'll comment it back out.